File: app/src/main/java/com/inspiredandroid/linuxcommandbibliotheca/interfaces/OnConvertFromHtmlToSpannableListener.java
package com.inspiredandroid.linuxcommandbibliotheca.interfaces;

import java.util.ArrayList;

/**
 * Created by Simon Schubert
 */
public interface OnConvertFromHtmlToSpannableListener {

    /**
     * @param spannable
     */
    void onConvertHtmlToSpannable(ArrayList<ArrayList<CharSequence>> spannable);
}

File: app/src/main/java/com/inspiredandroid/linuxcommandbibliotheca/interfaces/OnConvertFromHtmlToSpannableListener.kt
package com.inspiredandroid.linuxcommandbibliotheca.interfaces

import java.util.*

/**
 * Created by Simon Schubert
 */
interface OnConvertFromHtmlToSpannableListener {

    /**
     * @param spannable
     */
    fun onConvertHtmlToSpannable(spannable: ArrayList<ArrayList<CharSequence>>)
}
